src/main/java/com/indemnity83/irontank/utility/ItemMapHelper.java

```diff
@@ -1,49 +0,0 @@
-package com.indemnity83.irontank.utility;
-
-import java.util.HashMap;
-
-import com.indemnity83.irontank.init.ModBlocks;
-import com.indemnity83.irontank.init.ModItems;
-import net.minecraft.item.Item;
-import net.minecraft.block.Block;
-
-import cpw.mods.fml.common.event.FMLMissingMappingsEvent;
-
-public class ItemMapHelper {
-    private HashMap<String, Item> remapList = new HashMap();
-
-    public ItemMapHelper() {
-        remapList.put("irontank:ironGoldUpgrade", ModItems.ironGoldUpgrade);
-        remapList.put("irontank:goldDiamondUpgrade", ModItems.goldDiamondUpgrade);
-        remapList.put("irontank:silverDiamondUpgrade", ModItems.silverDiamondUpgrade);
-        remapList.put("irontank:copperSilverUpgrade", ModItems.copperSilverUpgrade);
-        remapList.put("irontank:silverGoldUpgrade", ModItems.silverGoldUpgrade);
-        remapList.put("irontank:copperIronUpgrade", ModItems.copperIronUpgrade);
-        remapList.put("irontank:glassIronUpgrade", ModItems.glassIronUpgrade);
-        remapList.put("irontank:glassCopperUpgrade", ModItems.glassCopperUpgrade);
-        remapList.put("irontank:diamondObsidianUpgrade", ModItems.diamondObsidianUpgrade);
-
-        remapList.put("irontank:copperTank", Item.getItemFromBlock(ModBlocks.copperTank));
-        remapList.put("irontank:ironTank", Item.getItemFromBlock(ModBlocks.ironTank));
-        remapList.put("irontank:silverTank", Item.getItemFromBlock(ModBlocks.silverTank));
-        remapList.put("irontank:goldTank", Item.getItemFromBlock(ModBlocks.goldTank));
-        remapList.put("irontank:obsidianTank", Item.getItemFromBlock(ModBlocks.obsidianTank));
-        remapList.put("irontank:diamondTank", Item.getItemFromBlock(ModBlocks.diamondTank));
-    }
-
-    public void handleMissingMaps(FMLMissingMappingsEvent event) {
```

Comment:
I still dont get why you removed this.

Reply:
This class is for not loss the items after the items id changes

Reply:
Now it is useless because I have reberted the items back